packages/EasyCodingStandard/src/Finder/DefaultFilesProvider.php

```diff
@@ -0,0 +1,54 @@
+<?php declare(strict_types=1);
+
+namespace Symplify\EasyCodingStandard\Finder;
+
+use SplFileInfo;
+use Symfony\Component\Finder\Finder;
+use Symplify\EasyCodingStandard\Contract\Finder\ExtraFilesProviderInterface;
+
+final class DefaultFilesProvider implements ExtraFilesProviderInterface
```

Comment:
All `ExtraFilesProviderInterface` instances [are added](https://github.com/Symplify/Symplify/blob/41dc8391d855f12cbc2acc93a7236488fcb5825c/packages/EasyCodingStandard/src/DependencyInjection/CompilerPass/CollectorCompilerPass.php#L57-L65) to `SourceFinder` (like all Commands are loaded to Symfony\Console\Application), so this one should not implement it.